devtools_m1/reset.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -e
+
+./docker_compose.sh exec -T -e MYSQL_PWD=magento db mysql -u magento magento -e "DROP DATABASE magento; CREATE DATABASE magento"
+./docker_compose.sh exec -T -e MYSQL_PWD=magento db mysql -u magento magento < db_data/dump.sql
+# ./docker_compose.sh exec -T -u www-data web /bin/bash -c "cd /var/www/html/magento/ && ./bin/magento cache:clean && ./bin/magento cache:flush"
```

Comment:
Is there a good reason for keeping this commented out line around... if there is, you could add a comment to say why.

Reply:
Heh. Thanks for keeping me honest. The reasoning is that I imagine I'll need to clear the cache as part of the reset process at some point, but I haven't encountered issues with it yet, so I haven't done so. I'll delete it.